composer.json

```diff
@@ -54,7 +54,7 @@
         "oat-sa/extension-tao-lti": "15.8.2",
         "oat-sa/extension-tao-ltideliveryprovider": "12.12.1",
         "oat-sa/extension-tao-revision": "10.4.0",
-        "oat-sa/extension-tao-mediamanager": "12.29.0",
+        "oat-sa/extension-tao-mediamanager": "12.29.2",
```

Comment:
In the version 12.29.1 we generated new bundles with different tao-core version, so this update will cause JS issues

Reply:
It should not be a problem. The code did not change, so the bundles are equivalent. The only difference is that meanwhile, the drop of IE support was effective in the bundler toolchain. Comparing both the commits, the only differences are in the way the code was transpiled.

Reply:
But `12.29.1` was generated using some tao-core from September release and the code in tao-community is from august release. I am not sure this is safe.

Reply:
https://github.com/oat-sa/extension-tao-mediamanager/pull/418

Reply:
Updated composer to use the new MediaManager backport: 6878b5fee418e067ec02d726d5cc7ee943c62878

I'll test the taoUpdate procedure next